content/docs/tasks/security/authn-policy/index.md

```diff
@@ -552,12 +544,28 @@ $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
 
 Attaching the valid token generated above returns success:
 
-{{< text bash >}}
+{{< text bash>}}
+$ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/demo.jwt -s)
 $ curl --header "Authorization: Bearer $TOKEN" $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
 200
 {{< /text >}}
 
-You may want to try to modify token or policy (e.g change issuer, audiences, expiry date etc) to observe other aspects of JWT validation.
+You may want to use the script [`gen-jwt.py`]({{< github_tree >}}/security/tools/jwt/samples) to generate new tokens for testing with different issuer, audiences, expiry date etc to observe other aspects of JWT validation. For example, the command below creates a token that will expire in 5 seconds. As you will see, requests using that token will be authenticated successfully at first, but then be rejected after 5 seconds:
```

Comment:
Watch out for the line length. My understanding is that 80 characters is the correct length for each line.

Reply:
Avoid using conjunctives and future tense. Writing only on the present tense helps readers identify with the content and gives the text a more active tone. Furthermore, it makes the text clearer and a lot easier to understand. I suggest these edits:
To observe other aspects of JWT validation, use the script [`gen-jwt.py`]({{< github_tree >}}/security/tools/jwt/samples) to generate new tokens to test with different issuer, audiences, expiry date, etc. For example, the command below creates a token that expires in 5 seconds. As you see, Istio authenticates requests using that token successfully at first but rejects them after 5 seconds:

Reply:
Done.

Btw, I use 160 for line limit. Do you want to change it to 80?